src/store/tests/test-data/cat-b/vehicle-checks/vehicle-checks.actions.ts

```diff
@@ -0,0 +1,50 @@
+import { createAction } from '@ngrx/store';
+import { VehicleChecksQuestion } from '../../../../../app/providers/question/vehicle-checks-question.model';
```

Comment:
can be changed to '@providers...' with new path aliases

Reply:
changed